docs/README.md

```diff
@@ -40,7 +40,7 @@ make build
 ```
 
 ```
-wmcb initialize-kubelet --ignition-file $IGNITION_FILE_PATH --kubelet-path $KUBELET_PATH
+wmcb initialize-kubelet --cluster-dns-ip $CLUSTER_DNS_IP --ignition-file $IGNITION_FILE_PATH --kubelet-path $KUBELET_PATH
```

Comment:
You need to document how to get the value for this flag as its not immediately obvious

Reply:
Additionally it would be nice to make it optional as suggested above